tests/ChatTests/Profiles.hs

```diff
@@ -3188,12 +3187,12 @@ testShortLinkInvitationSetIncognito =
       bob ##> ("/_prepare contact 1 " <> fullLink <> " " <> shortLink <> " " <> contactSLinkData)
       bob <## (aliceIncognito <> ": contact is prepared")
       bob ##> "/_connect contact @2 text hello"
-      _ <- getTermLine alice
       bob
         <### [ ConsoleString (aliceIncognito <> ": connection started"),
                WithTime ("@" <> aliceIncognito <> " hello")
              ]
       alice ?<# "bob> hello"
+      _ <- getTermLine alice -- what is that
```

Comment:
Probably incognito profile, why it moved?

Reply:
I don't know, it doesn't work otherwise